### PR TITLE
Update ownership

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Note `XliffTasks` does not force the items into a sorted order if they are not a
 
 ## Contact
 
-For more information, contact @tmeschter or @nguerrera on GitHub, or file an issue.
+For more information, contact @dotnet/dnceng on GitHub, or file an issue.
```
Comment (MattGal) [inline README.md:67]: for this link to work it needs to be `@dnceng`.  Let's wait til the afternoon though to make any such changes please.

**Comment by markwilkie** [comment]
Yea, confirming that we (Shawn's team) owns maintaining Xliff these days.

cc/ @shawnro

File: README.md (hunk 1 above)
Comment (MattGal) [reply README.md:67]: OK Mark confirmed, but please make the change so the at-tag works.

Comment (dagood) [reply README.md]: I don't understand this--I always see and have to type `@dotnet/dnceng`, `@dnceng` doesn't seem to do anything. (And either way, it's not a link in GitHub Markdown.) Is there a some other context I'm not thinking of?

Comment (MattGal) [reply README.md]: You're right, my mistake.  It doesn't render right in the "view" of the PR.

Comment (dagood) [reply README.md:67]: It looks like github flavored markdown uses `@` for emails, which is why they showed up highlighted originally... but yeah, doesn't mean anything here with nothing before the `@`.

**Pull request merged**